Makefile

```diff
@@ -40,7 +40,8 @@ $(BINS): vendor
 build-docker-bins: $(addsuffix -docker-bin,$(BINS))
```

Comment:
Do we want to build `linux-arm64` `brig` binaries as well?  If so, perhaps we add a section in the `%-cross-compile` target along the lines of:
```
if [ $$os = 'linux' ]; then \
  GOOS=$$os GOARCH=arm64 CGO_ENABLED=0 go build -ldflags '$(LDFLAGS)' -o ./bin/$*-$$os-arm64 ./$*/cmd/$*; \
fi; \
```

to build this arch along with the default/amd64?

Reply:
Do you mean in the `$(BINS): vendor` target?

Reply:
Oh good clarification... I was thinking the `%-cross-compile` target.  It's not super clear in the Makefile, but it's only currently used for building the brig cli binaries (which we upload with the GitHub release in CI on a tag push).  It's called by:
```
.PHONY: build-release
build-release: brig-cross-compile
```

Reply:
But only really necessary if we think the new `arm64`-variant `brig` binaries might be useful and should join the pre-existing amd64 linux, windows and darwin binaries.